src/isEmpty.js

```diff
@@ -12,10 +12,13 @@ var _curry1 = require('./internal/_curry1');
  * @return {Boolean}
  * @example
  *
- *      R.isEmpty([1, 2, 3]); //=> false
- *      R.isEmpty([]); //=> true
- *      R.isEmpty(''); //=> true
- *      R.isEmpty(null); //=> false
+ *      R.isEmpty([1, 2, 3]);   //=> false
+ *      R.isEmpty([]);          //=> true
+ *      R.isEmpty('');          //=> true
+ *      R.isEmpty(null);        //=> false
+ *      R.isEmpty(R.keys({}));  //=> true
+ *      R.isEmpty({});          //=> false ({} does not have a length property)
+ *      R.isEmpty({length: 0}); //=> true
```

Comment:
> I don't think of strings as array-like.

now that more & more list functions are being used for strings, we may have to revisit that question. at present, String is excluded from `isArrayLike`